<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background:rgb(255, 255, 255)
}</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="bieudo" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>1500</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboMode">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Offline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Online</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Host</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboMach">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Mạch1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mạch2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mạch3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mạch4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mạch5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>TP1.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="CH1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
color:rgb(255, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>Kênh Ch1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CH2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
color:rgb(0, 0, 255)
}</string>
     </property>
     <property name="text">
      <string>Kênh Ch2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboxung">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Sin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Răng Cưa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vuông</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="ZoomCh1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="PositionCh1">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ZoomCh2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="PositionCh2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Setting">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>80</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cài Đặt</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>1500</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Status_Arduino">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>30</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Status_Osi">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>60</y>
       <width>130</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Status_Twin">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>90</y>
       <width>150</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Run</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Trạng thái:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Mạch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Switch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>30</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Xung vào</string>
     </property>
    </widget>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>100</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>250</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Point</string>
     </property>
    </widget>
    <zorder>comboMode</zorder>
    <zorder>comboMach</zorder>
    <zorder>comboInput</zorder>
    <zorder>CH2</zorder>
    <zorder>comboxung</zorder>
    <zorder>ZoomCh1</zorder>
    <zorder>PositionCh1</zorder>
    <zorder>ZoomCh2</zorder>
    <zorder>PositionCh2</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>Setting</zorder>
    <zorder>frame_2</zorder>
    <zorder>Status_Arduino</zorder>
    <zorder>Status_Osi</zorder>
    <zorder>Status_Twin</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>dial</zorder>
    <zorder>label_10</zorder>
    <zorder>CH1</zorder>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Biểu đồ điện áp theo thời gian</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
